packages/components/psammead-media-player/src/index.stories.jsx

```diff
@@ -18,6 +18,22 @@ storiesOf('Components|Media Player', module).add('Default', () => (
   />
 ));
 
+storiesOf('Components|Media Player', module).add(
+  'Default with guidance',
```

Comment:
This is only a presentational suggestion, but most of our variations from default don't contain the word `default` in the story. If you prune `Default with` and just used `Guidance` it would also prevent the wrapping that's occurring.